Library/Formula/proxmark3.rb

```diff
@@ -0,0 +1,39 @@
+class Proxmark3 < Formula
+  desc "Proxmark3 client, flasher, HID flasher and firmware bundle"
+  homepage "http://www.proxmark.org"
+  url "https://github.com/Proxmark/proxmark3/archive/v2.2.0.tar.gz"
+  sha256 "e8f41fd36118d0569a02c1d2c50388238b11decd7ee526ba882c2b50b3f90069"
+  head "https://github.com/Proxmark/proxmark3.git"
+
+  depends_on "automake" => :build
+  depends_on "readline"
+  depends_on "p7zip" => :build
+  depends_on "libusb"
+  depends_on "libusb-compat"
+  depends_on "pkg-config" => :build
+  depends_on "wget"
+  depends_on "nitsky/stm32/arm-none-eabi-gcc" => :build
```

Comment:
You can't have mandatory cross-tap dependencies, and in the core we probably won't support even optional cross-tap dependencies to an unofficial tap.